chart/linkerd-buoyant/values.yaml

```diff
@@ -0,0 +1,36 @@
+# -- The ID for your Buoyant Cloud account. Required.
+accountID:
+
+# -- The key for your Buoyant Cloud account. Required.
+accountKey:
+
+# -- The download key for your Buoyant Cloud account. Required.
+accountDownloadKey:
+
+# -- The name of your cluster, as it should appear in Buoyant Cloud. Required
+clusterName:
+
+# -- Namespace where the agent components will be deployed
+namespace: buoyant-cloud
+
+# -- Set to false when installing linkerd-buoyant in a custom namespace. If so,
+# make sure you add the required annotations and labels, as depicted in the
+# namespace.yaml template
+installNamespace: true
```

Comment:
any reason not to just hard-code this? we currently don't support running the agent in different namespaces.

Reply:
Its a niche usecase, but if we want to use the chart locally, we would want to opt out out outputting the namespace manifest as we should already have a namespace because of the dockerhost config being applied beforehand. Right?

Reply:
More importantly, when people in prod are given the namespace they can work in, and don't have access to (or don't want to write to) the `default` namespace, which is where Helm v3 will put its config secret if its not instructed to use an existing namespace (through `helm install --namespace xxx`)